src/debug.c

```diff
@@ -67,345 +74,3 @@ void InstallBacktraceHandlers(void)
-    (VoidFunc)PARENT_LVARS,
-    (VoidFunc)PLAIN_LIST,
-    (VoidFunc)PopPlist,
-    (VoidFunc)POS_LIST,
-    (VoidFunc)POW,
-    (VoidFunc)PROD,
-    (VoidFunc)PROF_FUNC,
-    (VoidFunc)PTR_BAG,
-    (VoidFunc)PushPlist,
-    (VoidFunc)QUO,
-    (VoidFunc)RANK_PPERM,
-    (VoidFunc)RANK_TRANS,
-    (VoidFunc)RegisterStatWithHook,
-    (VoidFunc)ResizeWordSizedBag,
-    (VoidFunc)SAFE_C_ELM_FLAGS,
-    (VoidFunc)SET_BAG_FLAG,
-    (VoidFunc)SET_BIT_BLIST,
-    (VoidFunc)SET_BODY_FUNC,
-    (VoidFunc)SET_CACHE_OPER,
-    (VoidFunc)SET_CHAR_VALUE,
-    (VoidFunc)SET_DEN_RAT,
-    (VoidFunc)SET_ELM_PLIST,
-    (VoidFunc)SET_ELM_PREC,
-    (VoidFunc)SET_ENABLED_ATTR,
-    (VoidFunc)SET_ENVI_FUNC,
-    (VoidFunc)SET_FEXS_FUNC,
-    (VoidFunc)SET_FLAG1_FILT,
-    (VoidFunc)SET_FLAG2_FILT,
-    (VoidFunc)SET_FLAGS_FILT,
-    (VoidFunc)SET_HDLR_FUNC,
-    (VoidFunc)SET_INC_RANGE,
-    (VoidFunc)SET_IS_FILTER,
-    (VoidFunc)SET_LEN_BLIST,
-    (VoidFunc)SET_LEN_PLIST,
-    (VoidFunc)SET_LEN_PREC,
-    (VoidFunc)SET_LEN_RANGE,
-    (VoidFunc)SET_LEN_STRING,
-    (VoidFunc)SET_LOW_RANGE,
-    (VoidFunc)SET_METHS_OPER,
-    (VoidFunc)SET_NAMS_FUNC,
-    (VoidFunc)SET_NARG_FUNC,
-    (VoidFunc)SET_NLOC_FUNC,
-    (VoidFunc)SET_NUM_RAT,
-    (VoidFunc)SET_OBJ_FLAG,
-    (VoidFunc)SET_PROF_FUNC,
-    (VoidFunc)SET_PTR_BAG,
-    (VoidFunc)SET_RNAM_PREC,
-    (VoidFunc)SET_SETTR_FILT,
-    (VoidFunc)SET_STOREDINV_PERM,
-    (VoidFunc)SET_TESTR_FILT,
-    (VoidFunc)SET_TYPE_COMOBJ,
-    (VoidFunc)SET_TYPE_DATOBJ,
-    (VoidFunc)SET_TYPE_OBJ,
-    (VoidFunc)SET_TYPE_POSOBJ,
-    (VoidFunc)SET_VAL_MACFLOAT,
-    (VoidFunc)SETTR_FILT,
-    (VoidFunc)SHALLOW_COPY_OBJ,
-    (VoidFunc)SHRINK_PLIST,
-    (VoidFunc)SHRINK_STRING,
-    (VoidFunc)SINT_CHAR,
-    (VoidFunc)SIZE_BAG_CONTENTS,
-    (VoidFunc)SIZE_BAG,
-    (VoidFunc)SIZE_INT,
-    (VoidFunc)SIZE_OBJ,
-    (VoidFunc)SIZE_PLEN_BLIST,
-    (VoidFunc)SIZEBAG_PERM2,
-    (VoidFunc)SIZEBAG_PERM4,
-    (VoidFunc)SIZEBAG_STRINGLEN,
-    (VoidFunc)STAT_LVARS_PTR,
-    (VoidFunc)STAT_LVARS,
-    (VoidFunc)StateSlotsAtOffset,
-    (VoidFunc)STOREDINV_PERM,
-    (VoidFunc)SUM,
-    (VoidFunc)SWITCH_TO_OLD_LVARS_AND_FREE,
-    (VoidFunc)SWITCH_TO_OLD_LVARS,
-    (VoidFunc)SwitchToNewLvars,
-    (VoidFunc)TEST_BAG_FLAG,
-    (VoidFunc)TEST_BIT_BLIST,
-    (VoidFunc)TEST_OBJ_FLAG,
-    (VoidFunc)TESTR_FILT,
-    (VoidFunc)TNAM_OBJ,
-    (VoidFunc)TNUM_BAG,
-    (VoidFunc)TNUM_OBJ,
-    (VoidFunc)TYPE_COMOBJ,
-    (VoidFunc)TYPE_DATOBJ,
-    (VoidFunc)TYPE_OBJ,
-    (VoidFunc)TYPE_POSOBJ,
-    (VoidFunc)UNB_LIST,
-    (VoidFunc)UNB_REC,
-    (VoidFunc)VAL_MACFLOAT,
-    (VoidFunc)VisitStatIfHooked,
-    (VoidFunc)ZERO_MUT,
-    (VoidFunc)ZERO,
-#ifdef HPCGAP
-    (VoidFunc)ATOMIC_ELM_PLIST,
-    (VoidFunc)ATOMIC_SET_ELM_PLIST_ONCE,
-    (VoidFunc)ATOMIC_SET_ELM_PLIST,
-    (VoidFunc)GetTLS,
-    (VoidFunc)LCKS_FUNC,
-    (VoidFunc)REGION,
-    (VoidFunc)SET_LCKS_FUNC,
-    (VoidFunc)SET_REGION,
-#endif
-};
```

Comment:
This is one of the few changes which are not a result of "search and replace `static inline`".